docker-compose.dev.yml

```diff
@@ -100,3 +101,17 @@ services:
             - redis
             - clickhouse
             - kafka
+            - object_storage
+
+    object_storage:
+        image: minio/minio
+        ports:
+            - '19000:19000'
+            - '19001:19001'
+        volumes:
```

Comment:
For dev compose files, could we remove these volumes? I believe the Dockerfile already specifies that /data is a volume and should create an anonymous volume for us? Otherwise we might need to add `./object_storage` to gitignore

Reply:
object_storage is in ./gitignore already... but no point having things in the docker-compose file that are unnecessary? 👍